scripts/gen-aliasarr.template.sh

```
@@ -1881,141 +1864,94 @@ arr.vpn.status() {
     fi
   fi
 
-  local vpn_type pf_setting provider_key
-  vpn_type="${VPN_TYPE:-$(_arr_env_get VPN_TYPE 2>/dev/null || printf '')}"
-  vpn_type="$(_arr_lowercase "$vpn_type")"
-  if [ -z "$vpn_type" ]; then
-    vpn_type="openvpn"
+  local status_file="$(_arr_port_guard_status_file)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)